auth/build.gradle

```diff
@@ -1,13 +1,14 @@
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion project.ext.targetSdkVersion
+    compileSdkVersion 27
 
     defaultConfig {
         minSdkVersion project.ext.minSdkVersion
         targetSdkVersion project.ext.targetSdkVersion
 
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        android.defaultConfig.manifestPlaceholders = ['appAuthRedirectScheme': 'Invalid']
```

Comment:
Why we need it?

Reply:
It was breaking stuff for production, but placeholder is needed for testing local usage. Without that build fails.